docs/content/guides/7.multistore/3.patterns/5.subpath/1.subpath.md

```diff
@@ -0,0 +1,819 @@
+
+1. **Path-Based Configuration**
+    - Different configuration ID set by different paths allow to fetch data from different sources
+    - Example: `/apparel` path shows apparel products, while `/electronics` shows electronics products
+
+2. **Configuration ID vs Store ID**
+    - Each configuration has its own distinct ID
+    - This is different from the store ID (`storeId`)
+    - Multiple configurations can exist within a single store
+
+3. **Same-Store Operation**
+    - Config Switcher works within a single store instance
+    - It extracts the configuration ID from the request
+    - It executes requests using the appropriate configuration
+
+::tip Config Switcher Documentation
+This section provides a brief overview of setting up Config Switcher. For complete documentation, refer to the [Config Switcher guide](https://docs.alokai.com/middleware/guides/config-switcher).
+::
+
+### Key Distinctions To File-Based Inheritance
+
+`ConfigSwitcher` operates differently from file-based inheritance:
+
+| **File-Based Inheritance**              | **Config Switcher**                                              |
+| --------------------------------------- | ---------------------------------------------------------------- |
+| Creates separate deployable instances   | Uses a single deployable instance                                |
+| Provides inheritance mechanism to make managing multiple stores easy             | You have to manage codebase reuse between stores on your own |
+| Requires separate builds for each store | Single build supports multiple configurations                    |
+
+Take a look at the diagram to see the difference:
+
+<img src="../../img/file-based-vs-switcher-diagram.png" alt="Comparison of File-based Inheritance vs Config Switcher" class="mx-auto">
+
+## Setting Up the Middleware
+
+The first step is to configure your Middleware to support different configurations using the Config Switcher extension.
+
+Create a Config Switcher extension for your integration:
+
+```ts
+// apps/storefront-middleware/integrations/<integration-name>/extensions/configSwitcher.ts
+import { createConfigSwitcherExtension } from '@alokai/connect/config-switcher';
+import type { MiddlewareConfig } from '@vsf-enterprise/<integration>-api';
+
+export const configSwitcherExtension = createConfigSwitcherExtension<MiddlewareConfig>({
+  // Config switcher options - specify only what's different for each store
+  configuration: {
+    apparel: {
+      api: {
+        baseSiteId: 'apparel',
+        catalogId: 'apparelProductCatalog',
+      },
+    },
+    electronics: {
+      api: {
+        baseSiteId: 'electronics',
+        catalogId: 'electronicsProductCatalog',
+      },
+    },
+  },
+});
+```
+
+Then add this extension to your integration configuration:
+
+```ts
+// apps/storefront-middleware/integrations/<integration-name>/config.ts
+import { configSwitcherExtension } from './extensions/configSwitcher';
+
+export default {
```

Comment:
You are showing an example of an integration-specific config file but the example looks more like middleware.config.ts - I think that is what you meant:

```ts
export const config = {
  configuration: {},
  extensions: (extensions: ApiClientExtension[]) => [
    ...extensions,
  ],
  location: '@vsf-enterprise/<integration-name>/server',
} satisfies Integration<Config>;
```